pkg/controller/utils/errorscrub.go

```diff
@@ -2,11 +2,15 @@ package utils
 
 import (
 	"regexp"
+	"strings"
 )
 
 var (
+	newlineTabRE = regexp.MustCompile(`(\n\t)`)
```

Comment:
This isn't necessary. You're already chopping off at `\\r\\n` which will be earlier than `\n\t` (if that in fact exists in the string at all).

(Aside: the parens would be redundant anyway, since you're not using the capture group with `ReplaceAllString`.)

Reply:
This one's for scrubbing of AWS Request ID I think, but didn't end up needing any change

Reply:
Oh, just realized this was latent; and also that it is indeed necessary for some of the other message types. In a fup we could still:
```suggestion
	newlineTabRE = regexp.MustCompile(`\n\t`)
```